### ISSUE TITLE
Bug report: assets not loading behind proxy

### ISSUE THREAD

**Issue by mbande** [metabase/metabase#7841]
I've set up metabase v0.29.3 behind nginx with this config:
```
        location /mb/ {
                proxy_pass http://localhost:3000/;
        }
```
everything seems OK except icons in new question tab, which do not respect ```/mb/``` in path
![screenshot](https://user-images.githubusercontent.com/13609257/41202021-6780e13e-6cd8-11e8-8b15-c491d466be33.png)
also many other assets (css, js) are not loading

**Comment by jornh** [comment]
@mbande the fix mentioned above got released as part of v0.30.0. Would be good to know if that fixes this issue for you!

A good starting point for planning to go to v0.30 - as it contains some UI changes etc. is 
https://metabase.com/blog/before-upgrading-to-0.30/index.html#before-you-upgrade

**Comment by flamber** [comment]
Fixed in 0.30.0
